src/Pact/Types/Term.hs

```diff
@@ -90,8 +89,8 @@ import Pact.Types.Exp
 
 
 data DocModel = DocModel
```

Comment:
This is what we need to change back to `Meta`, right?

Reply:
I think I forgot the context here

Reply:
Also possible that I am getting things mixed up here -- `DocModel` used to be called `Meta`, and we were going to change it back, right?

Reply:
I wasn't planning to but if there's a good reason then sure!

Reply:
Found the earlier discussion: https://github.com/kadena-io/pact/pull/174#discussion_r206358566